### DISCUSSION TITLE
Why the second run is used for memory information?

### DISCUSSION THREAD

**Discussion by saeid93** [stanford-mast/INFaaS#7] [Q&A] [answered]
Hi INFaaS team,

I am using the profiler code for profiling some tensorflow model. I was wondering why the second run is used for memory information among all the three runs?
https://github.com/stanford-mast/INFaaS/blob/02ad2b86bcbc66d55b17baaef608c4e864b03918/src/profiler/profile_model.sh#L397

Thanks!

**Comment by faromero** [comment, marked as answer]
Hello,
This was simply done to ensure everything is loaded. Using a subsequent run would also work just fine.

Thanks for using INFaaS!

**Reply by saeid93** [reply]
Thanks!
